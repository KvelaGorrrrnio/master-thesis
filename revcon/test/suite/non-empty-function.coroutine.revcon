@coroutine

function nonEmptyLocalState() {
  local int a 5
}

coroutine main() {
  call @nonEmptyLocalState()
}
