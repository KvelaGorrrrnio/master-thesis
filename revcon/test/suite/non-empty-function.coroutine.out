The local store of @nonEmptyLocalState was expected to be empty, but has the following non-empty entries:
  a: 5
